2021-10-02 12:10:21,373 INFO    MainThread:51358 [wandb_setup.py:_flush():69] setting env: {}
2021-10-02 12:10:21,374 INFO    MainThread:51358 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-02 12:10:21,374 INFO    MainThread:51358 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211002_121021-3uu3kaeh/logs/debug.log
2021-10-02 12:10:21,374 INFO    MainThread:51358 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211002_121021-3uu3kaeh/logs/debug-internal.log
2021-10-02 12:10:21,374 INFO    MainThread:51358 [wandb_init.py:init():381] calling init triggers
2021-10-02 12:10:21,374 INFO    MainThread:51358 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-10-02 12:10:21,375 INFO    MainThread:51358 [wandb_init.py:init():430] starting backend
2021-10-02 12:10:21,375 INFO    MainThread:51358 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-02 12:10:21,381 INFO    MainThread:51358 [backend.py:ensure_launched():135] starting backend process...
2021-10-02 12:10:21,503 INFO    MainThread:51358 [backend.py:ensure_launched():140] started backend process with pid: 52672
2021-10-02 12:10:21,504 INFO    MainThread:51358 [wandb_init.py:init():435] backend started and connected
2021-10-02 12:10:21,506 INFO    MainThread:51358 [wandb_init.py:init():494] updated telemetry
2021-10-02 12:10:21,507 INFO    MainThread:51358 [wandb_init.py:init():517] communicating current version
2021-10-02 12:10:22,482 INFO    MainThread:51358 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-02 12:10:22,482 INFO    MainThread:51358 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-02 12:10:22,629 INFO    MainThread:51358 [wandb_init.py:init():557] starting run threads in backend
2021-10-02 12:10:27,637 INFO    MainThread:51358 [wandb_run.py:_console_start():1605] atexit reg
2021-10-02 12:10:27,637 INFO    MainThread:51358 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-10-02 12:10:27,638 INFO    MainThread:51358 [wandb_run.py:_redirect():1484] Redirecting console.
2021-10-02 12:10:27,642 INFO    MainThread:51358 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-02 12:10:27,642 INFO    MainThread:51358 [wandb_init.py:init():582] run started, returning control to user process
2021-10-02 12:10:58,862 INFO    MainThread:51358 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-10-02 12:10:58,862 INFO    MainThread:51358 [wandb_run.py:_restore():1547] restore
2021-10-02 12:11:01,650 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:11:02,190 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:11:02,319 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:11:02,690 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 24560
}

2021-10-02 12:11:02,801 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 24969
}

2021-10-02 12:11:02,905 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,008 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,111 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,213 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,330 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,587 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}

2021-10-02 12:11:03,865 INFO    MainThread:51358 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24969
  total_bytes: 24969
}
local_info {
}

2021-10-02 12:11:05,217 INFO    MainThread:51358 [wandb_run.py:_show_files():1947] logging synced files
